app/views/claimants_details/edit.html.slim

```diff
@@ -0,0 +1,61 @@
+/ TODO: Add required fields
+/ TODO: Check if there are "hidden" elements on the Jadu form
+.content-body.claimants-details
+  - content_for :form_page_title do
+    = "Claimant's Details"
```

Comment:
Only just noticed that we are not doing I18n translation - probably not checked this in previous PR's also.  Maybe do a jira ticket to do it retrospectively for all views ?  or go back and sort it now and do it as you work in future views - the choice is yours as long as we make sure its done by the end.

Reply:
I've made a ticket, RST-893.

My tests are running from an external content file though, in preparation.